public/app/plugins/datasource/loki/plugin.json

```
@@ -33,6 +33,14 @@
       {
         "name": "GitHub Project",
         "url": "https://github.com/grafana/loki"
+      },
+      {
+        "name": "Raise issue",
+        "url": "https://github.com/grafana/loki/issues/new"
```

```suggestion
        "url": "https://github.com/grafana/grafana/issues/new"
```

I think issues with the data source should still be raised in grafana/grafana.